custom_dc/setup_gcp.md

```diff
@@ -37,11 +32,22 @@ The installation can use an existing domain, or create a default domain
    Note: If this step fails, please contact support+custom@datacommons.org with
    the errors.
 
+1. [Optional] The default domain of the instance is
+   `<PROJECT_ID>-datacommons.com`. If you want to use an existing custom domain,
+   set the environment variable:
+
+   ```bash
+   export CUSTOM_DC_DOMAIN=<existing_domain>
+   ```
+
+   Later on, need to create a new DNS record by linking the domain with the IP
```

Comment:
nit: "you need to create..."

so its more obvious its on them?

Reply:
Done